### ISSUE TITLE
Fargate - platform version

### ISSUE THREAD

**Issue by vinsguru** [fabfuel/ecs-deploy#163]
Hi,

Is there any way to provide the platform version for the fargate deployment?

something like this?
--platform-version

**Comment by fabfuel** [comment]
Hi @vinsguru,

unfortunately this is currently not possible. I will tag the issue as a new feature request.

Best
Fabian

**Comment by vinsguru** [comment]
@fabfuel Thanks for your contribution. 
I just saw this - https://github.com/fabfuel/ecs-deploy/pull/135/commits/1faff926e42b37675bf663119d2beeb173b727da
Is the feature already there?

**Comment by fabfuel** [comment]
Yes, this is already available, but currently only for the `ecs run` command for running one-off tasks, not for `ecs deploy` to deploy a service. I'm actually not sure, if its possible to change the platform version for a service, do you know if this is possible?

Best
Fabian

**Comment by vinsguru** [comment]
Yes, I am able to change via aws-console for an ecs-service.

This also shows an option to updating the platform-version

https://docs.aws.amazon.com/cli/latest/reference/ecs/update-service.html
